inc/ThirdParty/Hostings/Cloudways.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Hostings;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Compatibility class for Cloudways Varnish
+ *
+ * @since 3.5.5
+ */
+class Cloudways implements Subscriber_Interface {
+	/**
+	 * Array of events this subscriber wants to listen to.
+	 *
+	 * @since 3.5.5
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		if ( ! isset( $_SERVER['cw_allowed_ip'] ) ) {
+			return [];
+		}
+
+		return [
+			'rocket_display_input_varnish_auto_purge' => 'return_false',
+			'do_rocket_varnish_http_purge'            => 'return_true',
+			'rocket_varnish_field_settings'           => 'varnish_addon_title',
+			'rocket_varnish_ip'                       => 'varnish_ip',
+		];
+	}
+
+	/**
+	 * Returns false
+	 *
+	 * @since 3.5.5
+	 *
+	 * @return bool
+	 */
+	public function return_false() {
+		return false;
+	}
+
+	/**
+	 * Returns true
+	 *
+	 * @since 3.5.5
+	 *
+	 * @return true
+	 */
+	public function return_true() {
+		return true;
+	}
```

Comment:
Why not return `__return_true` and `__return_false`?

Reply:
With the subscriber system currently, we can't use a global callback like those functions in `get_subscribed_events()`, it needs to be a method of the class.

Indeed I would have used those instead if possible to not re-invent the wheel.